crates/sui-graphql-rpc/src/data/pg.rs

```diff
@@ -66,6 +66,7 @@ impl QueryExecutor for PgExecutor {
         result.map_err(|e| Error::Internal(e.to_string()))
     }
 
+    /// TODO annotate with docs
```

Comment:
This is an implementation of a trait function so the doc comment is on the trait definition -- I think we can get rid of this comment?